kfs-core/src/main/java/edu/arizona/kfs/fp/document/DisbursementVoucherDocument.java

```diff
@@ -44,6 +44,9 @@
  * Document class override to ensure that the bank code is synchronized with the
  * payment method code.
  */
+// org.kuali.rice.kns.document.authorization.TransactionalDocumentPresentationController is deprecated
+// getReportable1099TransactionsFlag() and processAfterRetrieve() use unchecked Lists
+@SuppressWarnings({ "deprecation", "unchecked" })

```

Comment:
I'm not seeing a deprecation tag in TransactionalDocumentPresentationController, did I miss it? Also, for the "unchecked" suppression, I think limiting to the smallest scope is preferable, where less than 2-3 occurrences might be better to address on a line-by-line basis rather than the entire class (e.g. just line 352, 362 could be suppressed individually). Unless there is another change needed in this file though, I won't hold up a merge based on just this, but please do consider it in the future.